### ISSUE TITLE
GTA 3

### ISSUE THREAD

**Issue by Exarkuniv** [Exarkuniv/RetroPie-Extra#127]
https://github.com/jmcerrejon/PiKISS/blob/master/scripts/games/gta.sh

**Comment by Exarkuniv** [comment]
made a script, but cant get the correct files for the game to start, give up. if its that hard for me to find them all, it will be impossable for me to tell others to do it

**Comment by tacgnol3** [comment]
Can you put your script in the unstable repo ? i would like to try things to see if i can find a logic.

thanks

**Comment by Exarkuniv** [comment]
the one file i have not gotten yet is the gta.set. other then that i think i have them all

from what i can file is the config file, and the game will not start with out it.
and the only way i can figure out  to get that file is to get gta3 to run on your computer. which i cant get yet.

**Comment by Exarkuniv** [comment]
works with rasbain OS installed, games starts fine, but in retropie it fails to start
i get 2 errors
```
[DBG]: cdvd_stream: read info 0x1865c78
[DBG]: Using one streaming thread for all channels
[DBG]: Created cdstream thread
[DBG-2]: [../src/skel/glfw/glfw.cpp.psInitialize:484]: gGameState = GS_START_UP
casepath couldn't find dir/file "gta3.set", full path was gta3.set
[DBG-2]: Default skin set as no other skins are available OR saved skin not found!
[DBG]: Physical memory size 3838128128
[DBG]: Available physical memory 489484288
[DBG]: Shutting down CPedStats...
[DBG]: CPedStats shut down
xinit: connection to X server lost
```
```

[DBG]: cdvd_stream: read info 0x7e2c78
[DBG]: Using one streaming thread for all channels
[DBG]: Created cdstream thread
[DBG-2]: [../src/skel/glfw/glfw.cpp.psInitialize:484]: gGameState = GS_START_UP
casepath couldn't find dir/file "gta3.set", full path was gta3.set
[DBG-2]: Default skin set as no other skins are available OR saved skin not found!
[DBG]: Physical memory size 3838128128
[DBG]: Available physical memory 311271424
GLFW Error: GLX: Failed to create context: GLXBadFBConfig
OpenGL version: 2.1 Mesa 19.3.2
./re3: symbol lookup error: ./re3: undefined symbol: glfwUpdateGamepadMappings
xinit: connection to X server lost
```

**Comment by Exarkuniv** [comment]
Reopening. Might be able to get this. Might not but it's not the game files. Seems to only work in desktop environment. Need a workaround
